tsconfig.json

```diff
@@ -1,20 +1,17 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "noImplicitAny": true,
     "outDir": "./build/",
+    "skipLibCheck": true,
     "sourceMap": true,
-    "types": ["node"],
-    "target": "es6",
-    "noImplicitAny": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "target": "es2017",
```

Comment:
Whilst I don't expect it to be a problem, I would note that there are some ES2017 features that Node 8 doesn't support:
https://node.green/#ES2017

Is there a particular reason for specifying 2017 and not 2016?

Reply:
It matches what the cli specifies (so doesn't impose any additional restrictions) and enables generating native async/await statements rather than transpiling them

Reply:
I've checked the ES2017 features not supported by Node 8 in the node.green website, and I we're reasonably safe. Like Page mentioned, the CLI has been targeting ES2017 for about half a year by now.

Reply:
Fair enough, I'd never noticed before.